sql/sys_vars.cc

```diff
@@ -5579,3 +5579,15 @@ static Sys_var_charptr Sys_read_only_error_msg_extra(
        "which will be appended to read_only error messages.",
        GLOBAL_VAR(opt_read_only_error_msg_extra), CMD_LINE(OPT_ARG),
        IN_SYSTEM_CHARSET, DEFAULT(""), NO_MUTEX_GUARD, NOT_IN_BINLOG);
+
+static Sys_var_mybool Sys_log_legacy_user(
+       "log_legacy_user",
+       "Log legacy user names in slow query log.",
+       GLOBAL_VAR(log_legacy_user),
+       CMD_LINE(OPT_ARG), DEFAULT(FALSE));
+
+static Sys_var_charptr Sys_legacy_user_identifier(
+       "legacy_user_identifier",
+       "String that will identify the user name as a legacy user",
```

Comment:
Can you update this description to indicate that if the string is found _anywhere_ in the user name it will mark the user name as a legacy user?  It is unclear where this string needs to be as it is.  An example would be "Identifier which if found anywhere in the user name will identify the user name as a legacy user".